include/seqan3/range/container/small_vector.hpp

```diff
@@ -869,29 +869,20 @@ class small_vector
 
     //!\brief Performs element-wise comparison.
     template <size_t cap2>
-    //!\cond
-        requires cap2 <= capacity_ /* resolves ambiguousness when comparing two small_vectors of unequal capacity */
```

Comment:
I am pretty sure these used to be necessary. How does the compiler resolve the ambiguity now? Is the operator of the LHS or the RHS taken?

I mean it's great if we can remove it, just making sure.

Reply:
I don't think so, because requires on friends never worked up until gcc 9.

`small_vector` is a complete type in this case and there is no other type specialisation or overload for small_vector.

Reply:
> I don't think so, because requires on friends never worked up until gcc 10.

You are right of course!